src/main/resources/application.yml

```diff
@@ -1,6 +1,11 @@
 spring:
   redis:
-    host: ${redis.host}
+    host: your-hostname
```

Comment:
A default value of "localhost" would be useful for local development cases. You could also allow for a "short hand" config using `${redis.host:localhost}`